modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -230,6 +372,8 @@ Deallocation is currently prevented in the following situations.
 * If a quorum of servers hosting the database in primary mode are cordoned.
 ====
 
+Once the command has been executed, the server changes state to `deallocating` and it cannot readily be enabled again, see the earlier section, xref:clustering/servers.adoc#_dropped_state[Dropped state] for more information.
```

Comment:
```suggestion
Once the command has been executed, the server changes state to `deallocating` and it cannot readily be enabled again.
See xref:clustering/servers.adoc#_dropped_state[Dropped state] for more information.
```

Reply:
Since _Dropped state_ is on the same page, I feel that it should be made clear that the linked section is located above and not on a different page. For accessibility reasons, I decided against saying _See Dropped state above_.